backend/file_management/file_management_helper.py

```
@@ -81,9 +81,16 @@ def get_files(fs: AbstractFileSystem, file_path: str) -> list[FileInformation]:
             if os.path.normpath(file_path) == os.path.normpath(file_name):
                 continue
 
-            # Call fs.info() to get file size if its missing
+            # Backfill missing size via fs.info(). pydrive2 raises int(None) on
+            # Google-native docs (no fileSize); fall back to 0, don't fail the listing.
             if file_info.get("type") == "file" and file_info.get("size") is None:
-                file_info = fs.info(file_name)
+                try:
+                    file_info = fs.info(file_name)
+                except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)